Thu Mar 31 12:20:48 CEST 2022 : Apache Derby Network Server - 10.15.2.0 - (1873585) started and ready to accept connections on port 1527
----------------------------------------------------------------
Thu Mar 31 12:22:33 CEST 2022:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.15.2.0 - (1873585): instance a816c00e-017f-df7d-f71b-ffffc80ef227 
on database directory /home/valentin/Desktop/ITP/projekt/clone 4/02-project-versteigerungs-programm/Versteigerungs_Programm/db/db with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@14514713 
Loaded from file:/home/valentin/Desktop/db-derby-10.15.2.0-bin/lib/derby.jar
java.vendor=Private Build
user.dir=/home/valentin/Desktop/ITP/projekt/clone 4/02-project-versteigerungs-programm/Versteigerungs_Programm/db
os.name=Linux
os.arch=amd64
os.version=5.13.0-37-generic
derby.system.home=/home/valentin/Desktop/ITP/projekt/clone 4/02-project-versteigerungs-programm/Versteigerungs_Programm/db
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Thu Mar 31 12:22:38 CEST 2022:
Shutting down instance a816c00e-017f-df7d-f71b-ffffc80ef227 on database directory /home/valentin/Desktop/ITP/projekt/clone 4/02-project-versteigerungs-programm/Versteigerungs_Programm/db/db with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@14514713 
Thu Mar 31 12:22:38 CEST 2022 Thread[DRDAConnThread_2,5,main] Cleanup action starting
ERROR XBM01: Startup failed due to an exception. See next exception for details. 
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.iapi.services.monitor.Monitor.exceptionStartingModule(Monitor.java:743)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1857)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
Caused by: java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	... 22 more
============= begin nested exception, level (1) ===========
java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
============= end nested exception, level (1) ===========
Cleanup action completed
Thu Mar 31 12:22:38 CEST 2022 Thread[DRDAConnThread_2,5,main] Cleanup action starting
java.sql.SQLException: Failed to create database 'db', see the next exception for details.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:115)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.impl.jdbc.Util.seeNextException(Util.java:252)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2661)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
Caused by: ERROR XJ041: Failed to create database 'db', see the next exception for details.
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	... 18 more
Caused by: ERROR XBM01: Startup failed due to an exception. See next exception for details. 
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.iapi.services.monitor.Monitor.exceptionStartingModule(Monitor.java:743)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1857)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	... 15 more
Caused by: java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	... 22 more
============= begin nested exception, level (1) ===========
java.sql.SQLException: Startup failed due to an exception. See next exception for details. 
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:115)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.impl.jdbc.Util.seeNextException(Util.java:252)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(TransactionResourceImpl.java:438)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.handleException(TransactionResourceImpl.java:360)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleException(EmbedConnection.java:2405)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2662)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
Caused by: ERROR XBM01: Startup failed due to an exception. See next exception for details. 
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	... 21 more
Caused by: java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	... 15 more
============= end nested exception, level (1) ===========
============= begin nested exception, level (2) ===========
java.sql.SQLException: Java exception: 'access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH"): java.security.AccessControlException'.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:115)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.impl.jdbc.Util.seeNextException(Util.java:252)
	at org.apache.derby.impl.jdbc.Util.javaException(Util.java:274)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(TransactionResourceImpl.java:444)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(TransactionResourceImpl.java:439)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.handleException(TransactionResourceImpl.java:360)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleException(EmbedConnection.java:2405)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2662)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
Caused by: ERROR XJ001: Java exception: 'access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH"): java.security.AccessControlException'.
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	... 23 more
Caused by: java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	... 15 more
============= end nested exception, level (2) ===========
============= begin nested exception, level (3) ===========
ERROR XJ001: Java exception: 'access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH"): java.security.AccessControlException'.
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.impl.jdbc.Util.seeNextException(Util.java:252)
	at org.apache.derby.impl.jdbc.Util.javaException(Util.java:274)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(TransactionResourceImpl.java:444)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(TransactionResourceImpl.java:439)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.handleException(TransactionResourceImpl.java:360)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleException(EmbedConnection.java:2405)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2662)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
Caused by: java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	... 15 more
============= end nested exception, level (3) ===========
============= begin nested exception, level (4) ===========
java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
============= end nested exception, level (4) ===========
Cleanup action completed
Thu Mar 31 12:22:38 CEST 2022 Thread[DRDAConnThread_2,5,main] (DATABASE = db), (DRDAID = {1}), Failed to create database 'db', see the next exception for details.
Thu Mar 31 12:22:38 CEST 2022 Thread[DRDAConnThread_2,5,main] (DATABASE = db), (DRDAID = {1}), Startup failed due to an exception. See next exception for details. 
Thu Mar 31 12:22:38 CEST 2022 Thread[DRDAConnThread_2,5,main] (DATABASE = db), (DRDAID = {1}), Java exception: 'access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH"): java.security.AccessControlException'.
----------------------------------------------------------------
Thu Mar 31 12:24:11 CEST 2022:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.15.2.0 - (1873585): instance e66540fa-017f-df7d-f71b-ffffc80ef227 
on database directory /home/valentin/Desktop/ITP/projekt/clone 4/02-project-versteigerungs-programm/Versteigerungs_Programm/db/db with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@14514713 
Loaded from file:/home/valentin/Desktop/db-derby-10.15.2.0-bin/lib/derby.jar
java.vendor=Private Build
user.dir=/home/valentin/Desktop/ITP/projekt/clone 4/02-project-versteigerungs-programm/Versteigerungs_Programm/db
os.name=Linux
os.arch=amd64
os.version=5.13.0-37-generic
derby.system.home=/home/valentin/Desktop/ITP/projekt/clone 4/02-project-versteigerungs-programm/Versteigerungs_Programm/db
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Thu Mar 31 12:24:15 CEST 2022:
Shutting down instance e66540fa-017f-df7d-f71b-ffffc80ef227 on database directory /home/valentin/Desktop/ITP/projekt/clone 4/02-project-versteigerungs-programm/Versteigerungs_Programm/db/db with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@14514713 
Thu Mar 31 12:24:15 CEST 2022 Thread[DRDAConnThread_2,5,main] Cleanup action starting
ERROR XBM01: Startup failed due to an exception. See next exception for details. 
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.iapi.services.monitor.Monitor.exceptionStartingModule(Monitor.java:743)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1857)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
Caused by: java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	... 22 more
============= begin nested exception, level (1) ===========
java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
============= end nested exception, level (1) ===========
Cleanup action completed
Thu Mar 31 12:24:15 CEST 2022 Thread[DRDAConnThread_2,5,main] Cleanup action starting
java.sql.SQLException: Failed to create database 'db', see the next exception for details.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:115)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.impl.jdbc.Util.seeNextException(Util.java:252)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2661)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
Caused by: ERROR XJ041: Failed to create database 'db', see the next exception for details.
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	... 18 more
Caused by: ERROR XBM01: Startup failed due to an exception. See next exception for details. 
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.iapi.services.monitor.Monitor.exceptionStartingModule(Monitor.java:743)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1857)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	... 15 more
Caused by: java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	... 22 more
============= begin nested exception, level (1) ===========
java.sql.SQLException: Startup failed due to an exception. See next exception for details. 
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:115)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.impl.jdbc.Util.seeNextException(Util.java:252)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(TransactionResourceImpl.java:438)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.handleException(TransactionResourceImpl.java:360)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleException(EmbedConnection.java:2405)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2662)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
Caused by: ERROR XBM01: Startup failed due to an exception. See next exception for details. 
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	... 21 more
Caused by: java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	... 15 more
============= end nested exception, level (1) ===========
============= begin nested exception, level (2) ===========
java.sql.SQLException: Java exception: 'access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH"): java.security.AccessControlException'.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:115)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.impl.jdbc.Util.seeNextException(Util.java:252)
	at org.apache.derby.impl.jdbc.Util.javaException(Util.java:274)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(TransactionResourceImpl.java:444)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(TransactionResourceImpl.java:439)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.handleException(TransactionResourceImpl.java:360)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleException(EmbedConnection.java:2405)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2662)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
Caused by: ERROR XJ001: Java exception: 'access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH"): java.security.AccessControlException'.
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	... 23 more
Caused by: java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	... 15 more
============= end nested exception, level (2) ===========
============= begin nested exception, level (3) ===========
ERROR XJ001: Java exception: 'access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH"): java.security.AccessControlException'.
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.impl.jdbc.Util.seeNextException(Util.java:252)
	at org.apache.derby.impl.jdbc.Util.javaException(Util.java:274)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(TransactionResourceImpl.java:444)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(TransactionResourceImpl.java:439)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.handleException(TransactionResourceImpl.java:360)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleException(EmbedConnection.java:2405)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2662)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
Caused by: java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	... 15 more
============= end nested exception, level (3) ===========
============= begin nested exception, level (4) ===========
java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
============= end nested exception, level (4) ===========
Cleanup action completed
Thu Mar 31 12:24:15 CEST 2022 Thread[DRDAConnThread_2,5,main] (DATABASE = db), (DRDAID = {2}), Failed to create database 'db', see the next exception for details.
Thu Mar 31 12:24:15 CEST 2022 Thread[DRDAConnThread_2,5,main] (DATABASE = db), (DRDAID = {2}), Startup failed due to an exception. See next exception for details. 
Thu Mar 31 12:24:15 CEST 2022 Thread[DRDAConnThread_2,5,main] (DATABASE = db), (DRDAID = {2}), Java exception: 'access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH"): java.security.AccessControlException'.
----------------------------------------------------------------
Thu Mar 31 12:24:24 CEST 2022:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.15.2.0 - (1873585): instance 5b17c1e6-017f-df7d-f71b-ffffc80ef227 
on database directory /home/valentin/Desktop/ITP/projekt/clone 4/02-project-versteigerungs-programm/Versteigerungs_Programm/db/db with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@14514713 
Loaded from file:/home/valentin/Desktop/db-derby-10.15.2.0-bin/lib/derby.jar
java.vendor=Private Build
user.dir=/home/valentin/Desktop/ITP/projekt/clone 4/02-project-versteigerungs-programm/Versteigerungs_Programm/db
os.name=Linux
os.arch=amd64
os.version=5.13.0-37-generic
derby.system.home=/home/valentin/Desktop/ITP/projekt/clone 4/02-project-versteigerungs-programm/Versteigerungs_Programm/db
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Thu Mar 31 12:24:29 CEST 2022:
Shutting down instance 5b17c1e6-017f-df7d-f71b-ffffc80ef227 on database directory /home/valentin/Desktop/ITP/projekt/clone 4/02-project-versteigerungs-programm/Versteigerungs_Programm/db/db with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@14514713 
Thu Mar 31 12:24:29 CEST 2022 Thread[DRDAConnThread_2,5,main] Cleanup action starting
ERROR XBM01: Startup failed due to an exception. See next exception for details. 
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.iapi.services.monitor.Monitor.exceptionStartingModule(Monitor.java:743)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1857)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
Caused by: java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	... 22 more
============= begin nested exception, level (1) ===========
java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
============= end nested exception, level (1) ===========
Cleanup action completed
Thu Mar 31 12:24:29 CEST 2022 Thread[DRDAConnThread_2,5,main] Cleanup action starting
java.sql.SQLException: Failed to create database 'db', see the next exception for details.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:115)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.impl.jdbc.Util.seeNextException(Util.java:252)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2661)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
Caused by: ERROR XJ041: Failed to create database 'db', see the next exception for details.
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	... 18 more
Caused by: ERROR XBM01: Startup failed due to an exception. See next exception for details. 
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.iapi.services.monitor.Monitor.exceptionStartingModule(Monitor.java:743)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1857)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	... 15 more
Caused by: java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	... 22 more
============= begin nested exception, level (1) ===========
java.sql.SQLException: Startup failed due to an exception. See next exception for details. 
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:115)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.impl.jdbc.Util.seeNextException(Util.java:252)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(TransactionResourceImpl.java:438)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.handleException(TransactionResourceImpl.java:360)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleException(EmbedConnection.java:2405)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2662)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
Caused by: ERROR XBM01: Startup failed due to an exception. See next exception for details. 
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	... 21 more
Caused by: java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	... 15 more
============= end nested exception, level (1) ===========
============= begin nested exception, level (2) ===========
java.sql.SQLException: Java exception: 'access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH"): java.security.AccessControlException'.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:115)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.impl.jdbc.Util.seeNextException(Util.java:252)
	at org.apache.derby.impl.jdbc.Util.javaException(Util.java:274)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(TransactionResourceImpl.java:444)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(TransactionResourceImpl.java:439)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.handleException(TransactionResourceImpl.java:360)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleException(EmbedConnection.java:2405)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2662)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
Caused by: ERROR XJ001: Java exception: 'access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH"): java.security.AccessControlException'.
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	... 23 more
Caused by: java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	... 15 more
============= end nested exception, level (2) ===========
============= begin nested exception, level (3) ===========
ERROR XJ001: Java exception: 'access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH"): java.security.AccessControlException'.
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.impl.jdbc.Util.seeNextException(Util.java:252)
	at org.apache.derby.impl.jdbc.Util.javaException(Util.java:274)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(TransactionResourceImpl.java:444)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(TransactionResourceImpl.java:439)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.handleException(TransactionResourceImpl.java:360)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleException(EmbedConnection.java:2405)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2662)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
Caused by: java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	... 15 more
============= end nested exception, level (3) ===========
============= begin nested exception, level (4) ===========
java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
============= end nested exception, level (4) ===========
Cleanup action completed
Thu Mar 31 12:24:29 CEST 2022 Thread[DRDAConnThread_2,5,main] (DATABASE = db), (DRDAID = {3}), Failed to create database 'db', see the next exception for details.
Thu Mar 31 12:24:29 CEST 2022 Thread[DRDAConnThread_2,5,main] (DATABASE = db), (DRDAID = {3}), Startup failed due to an exception. See next exception for details. 
Thu Mar 31 12:24:29 CEST 2022 Thread[DRDAConnThread_2,5,main] (DATABASE = db), (DRDAID = {3}), Java exception: 'access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH"): java.security.AccessControlException'.
----------------------------------------------------------------
Thu Mar 31 12:24:29 CEST 2022:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.15.2.0 - (1873585): instance 062e42d2-017f-df7d-f71b-ffffc80ef227 
on database directory /home/valentin/Desktop/ITP/projekt/clone 4/02-project-versteigerungs-programm/Versteigerungs_Programm/db/db with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@14514713 
Loaded from file:/home/valentin/Desktop/db-derby-10.15.2.0-bin/lib/derby.jar
java.vendor=Private Build
user.dir=/home/valentin/Desktop/ITP/projekt/clone 4/02-project-versteigerungs-programm/Versteigerungs_Programm/db
os.name=Linux
os.arch=amd64
os.version=5.13.0-37-generic
derby.system.home=/home/valentin/Desktop/ITP/projekt/clone 4/02-project-versteigerungs-programm/Versteigerungs_Programm/db
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Thu Mar 31 12:24:34 CEST 2022:
Shutting down instance 062e42d2-017f-df7d-f71b-ffffc80ef227 on database directory /home/valentin/Desktop/ITP/projekt/clone 4/02-project-versteigerungs-programm/Versteigerungs_Programm/db/db with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@14514713 
Thu Mar 31 12:24:34 CEST 2022 Thread[DRDAConnThread_2,5,main] Cleanup action starting
ERROR XBM01: Startup failed due to an exception. See next exception for details. 
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.iapi.services.monitor.Monitor.exceptionStartingModule(Monitor.java:743)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1857)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
Caused by: java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	... 22 more
============= begin nested exception, level (1) ===========
java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
============= end nested exception, level (1) ===========
Cleanup action completed
Thu Mar 31 12:24:34 CEST 2022 Thread[DRDAConnThread_2,5,main] Cleanup action starting
java.sql.SQLException: Failed to create database 'db', see the next exception for details.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:115)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.impl.jdbc.Util.seeNextException(Util.java:252)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2661)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
Caused by: ERROR XJ041: Failed to create database 'db', see the next exception for details.
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	... 18 more
Caused by: ERROR XBM01: Startup failed due to an exception. See next exception for details. 
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.iapi.services.monitor.Monitor.exceptionStartingModule(Monitor.java:743)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1857)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	... 15 more
Caused by: java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	... 22 more
============= begin nested exception, level (1) ===========
java.sql.SQLException: Startup failed due to an exception. See next exception for details. 
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:115)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.impl.jdbc.Util.seeNextException(Util.java:252)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(TransactionResourceImpl.java:438)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.handleException(TransactionResourceImpl.java:360)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleException(EmbedConnection.java:2405)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2662)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
Caused by: ERROR XBM01: Startup failed due to an exception. See next exception for details. 
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	... 21 more
Caused by: java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	... 15 more
============= end nested exception, level (1) ===========
============= begin nested exception, level (2) ===========
java.sql.SQLException: Java exception: 'access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH"): java.security.AccessControlException'.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:115)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.impl.jdbc.Util.seeNextException(Util.java:252)
	at org.apache.derby.impl.jdbc.Util.javaException(Util.java:274)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(TransactionResourceImpl.java:444)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(TransactionResourceImpl.java:439)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.handleException(TransactionResourceImpl.java:360)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleException(EmbedConnection.java:2405)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2662)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
Caused by: ERROR XJ001: Java exception: 'access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH"): java.security.AccessControlException'.
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	... 23 more
Caused by: java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	... 15 more
============= end nested exception, level (2) ===========
============= begin nested exception, level (3) ===========
ERROR XJ001: Java exception: 'access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH"): java.security.AccessControlException'.
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:300)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.impl.jdbc.Util.seeNextException(Util.java:252)
	at org.apache.derby.impl.jdbc.Util.javaException(Util.java:274)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(TransactionResourceImpl.java:444)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(TransactionResourceImpl.java:439)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.handleException(TransactionResourceImpl.java:360)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleException(EmbedConnection.java:2405)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2662)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
Caused by: java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	... 15 more
============= end nested exception, level (3) ===========
============= begin nested exception, level (4) ===========
java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1036)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:408)
	at java.base/java.lang.System.getenv(System.java:1016)
	at java.base/java.util.Properties.getFormattedTimestamp(Properties.java:1599)
	at java.base/java.util.Properties.store0(Properties.java:926)
	at java.base/java.util.Properties.store(Properties.java:915)
	at org.apache.derby.impl.services.monitor.StorageFactoryService$5.run(StorageFactoryService.java:369)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.services.monitor.StorageFactoryService.saveServiceProperties(StorageFactoryService.java:347)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(BaseMonitor.java:1840)
	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(BaseMonitor.java:1022)
	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(FileMonitor.java:48)
	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Monitor.java:650)
	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(EmbedConnection.java:4067)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(EmbedConnection.java:4061)
	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(EmbedConnection.java:2654)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:435)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:660)
	at org.apache.derby.iapi.jdbc.InternalDriver$1.run(InternalDriver.java:656)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:654)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:300)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:967)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1476)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1426)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3380)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1210)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1010)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:300)
============= end nested exception, level (4) ===========
Cleanup action completed
Thu Mar 31 12:24:34 CEST 2022 Thread[DRDAConnThread_2,5,main] (DATABASE = db), (DRDAID = {4}), Failed to create database 'db', see the next exception for details.
Thu Mar 31 12:24:34 CEST 2022 Thread[DRDAConnThread_2,5,main] (DATABASE = db), (DRDAID = {4}), Startup failed due to an exception. See next exception for details. 
Thu Mar 31 12:24:34 CEST 2022 Thread[DRDAConnThread_2,5,main] (DATABASE = db), (DRDAID = {4}), Java exception: 'access denied ("java.lang.RuntimePermission" "getenv.SOURCE_DATE_EPOCH"): java.security.AccessControlException'.
